assets/js/inventory.js

```diff
@@ -15,10 +16,16 @@ var Inventory = {
       $.cookie('inventory-menu-update-enabled', '1', { expires: 999 });
     }
 
+    if (typeof $.cookie('reset-changing-inventory-enabled') === 'undefined') {
+      Inventory.resetButtonChangingInventory = false;
+      $.cookie('reset-changing-inventory-enabled', '0', { expires: 999 });
+    }
+
```

Comment:
Make `true` and `1` the default so default behavior doesn't change.

Reply:
i wanted to change this because that option was sometimes also annoying for me